### ISSUE TITLE
Accuracy 

### ISSUE THREAD

**Issue by zsmhlubi** [makavelli3145/ML-project-2023#2]
In the algorithm I tried using sklearn and got up to 45% accuracy but am struggling to get it to do better and I had a question in terms of the data is it possible that my algorithm is overfitting the model which in turn make it do wrong prediction sometimes
